conf/application.conf

```diff
@@ -146,6 +146,10 @@ signOut {
   url = "http://localhost:9553/bas-gateway/sign-out-without-state?continue="
 }
 
+whatYouOwe {
+  url = "http://localhost:9152/vat-through-software/what-you-owe"
```

Comment:
This will need an app-config base PR to override in envs

Reply:
app config base pr created and config moved to urls section
https://github.com/hmrc/app-config-base/pull/7771